WordPress/WordPressTest/ReaderPostServiceTest.m

```diff
@@ -36,7 +36,7 @@ - (RemoteReaderPost *)remoteReaderPostForTests {
 - (void)testDeletePostsWithoutATopic {
     id<CoreDataStack> coreDataStack = [self coreDataStackForTesting];
     NSManagedObjectContext *context = [coreDataStack mainContext];
-    ReaderPostService *service = [[ReaderPostService alloc] initWithManagedObjectContext:context];
+    ReaderPostService *service = [[ReaderPostService alloc] initWithCoreDataStack:coreDataStack];
```

Comment:
Out of scope, sort of. I was surprised this was the only change in the tests. Then looked at the tests and saw this is _the only test_ for `ReaderPostService`. The related `ReaderTopicService` and `ReaderSiteService` don't have tests at all 😞 

I wonder when/if we should add coverage for them.

Reply:
I agree more unit test coverage is definitely better. Even super basic ones like asserting the success or failure block is called would be useful. I can keep a note to myself and create a follow up PR to add some tests.

Reply:
That sounds like a great task to keep for the "end" of the project.

Reply:
`ReaderTopicService` is covered in [`ReaderTopicServiceTest`](https://github.com/wordpress-mobile/WordPress-iOS/blob/6453e6efb3a2972805b5823a0ae1a992e3e5090c/WordPress/WordPressTest/ReaderTopicServiceTest.swift), so I've created [a PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/20216) to add some unit tests for `ReaderSiteService`.